modules/caddyhttp/matchers_test.go

```diff
@@ -518,3 +518,12 @@ func TestResponseMatcher(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkHostMatcherWithoutPlaceholders(b *testing.B) {
+	req := &http.Request{Host: "localhost"}
+	match := MatchHost{"localhost"}
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		match.Match(req)
+	}
+}
```

Comment:
It would be interesting to also benchmark when a replacement happens. i.e. use a placeholder in `MatchHost` instead of `"localhost"`.